src/index.ts

```diff
@@ -209,26 +203,35 @@ class Storyblok {
 		per_page: number,
 		page: number
 	): Promise<ISbResult> {
-		const options = this.factoryParamOptions(
+		const query = this.factoryParamOptions(
 			url,
 			this.helpers.getOptionsPage(params, per_page, page)
 		)
 
-		return this.cacheResponse(url, options)
+		return this.cacheResponse(url, query)
 	}
 
-	public get(slug: string, params?: ISbStoriesParams): Promise<ISbResult> {
+	public get(
+		slug: string,
+		params?: ISbStoriesParams,
+		fetchOptions?: RequestInit
+	): Promise<ISbResult> {
 		if (!params) params = {} as ISbStoriesParams
 		const url = `/${slug}`
 		const query = this.factoryParamOptions(url, params)
 
+		if (fetchOptions) {
+			this.client.setFetchOptions(fetchOptions)
```

Comment:
@thiagosaife the fetch options must happen per-call. If we store them in the instance, we must make sure that we clean them before next call.

Otherwise this scenario can happen:
- 1: User calls `.get(url, sbOptions, { customParam: 1 })`
- 2: User calls `.get(url, sbOptions)` (bad - fetch will happen with { customParam: 1} as it's not cleaned in the instance)